_posts/2019-08-16-the-future-of-iot.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+current: post
+cover: 2019-08-16-the-future-of-iot.png
+title: The Future of IoT
+date: 2019-08-16 13:00:00
+category: [news,thoughts]
+author: Nikoletta Triantafyllopoulou
+tags: [Open Source]
+class: post-template
+---
+{% asset blog/2019-08-16-the-future-of-iot.png srcset:width="1300 2x" srcset:width="650 1x" alt="IoTSecurityIssues" %}
```

Comment:
No - we don't want the image to be *both* in the header *and* at the top of the page. That's redundant and looks weird.

Reply:
@vpetersson fixed this